.changeset/sharp-dryers-happen.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin": minor
+---
+
+Fix: Remove `display: flex` from `children`-slot of `FieldSet` to allow non-full-width fields
```

Comment:
```suggestion
"@comet/admin": patch
---

Remove `display: flex` from `children`-slot of `FieldSet` to allow non-full-width fields
```

Reply:
```suggestion
"@comet/admin": patch
---

Allow non-full-width fields in `FieldSet`
```

I'd even remove the implementation details from the changeset 😁 But I'm fine with @jamesricky's suggestion too!